app/views/users/show.html.haml

```diff
@@ -71,20 +71,19 @@
               %p
                 %span{style: "font-weight: normal; font-size: 9pt; padding-left: 7px;"}
                   %a#edit_custom_ontologies{href: "javascript:void(0);"}= t('users.show.select_semantic_resources')
```

Comment:
as we are here, migrate the old Jquery  code related to this to a new Stimulus controller, or use an existing one, as reveal component
![image](https://github.com/ontoportal-lirmm/bioportal_web_ui/assets/29259906/a5c3f589-764f-4e9f-9957-ed3ea59cb7f0)